app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendGeographicMessageDialog.java

```diff
@@ -125,18 +157,32 @@ protected boolean hasPositiveButton() {
     protected void onPositiveClick() {
         sLogger.userInteraction("Clicked OK");
 
-        mMessageSender.sendLatLongMessageAsync(mPoint);
+        if (!validateDialogInput()) {
```

Comment:
To make it more clear, put the logic lines inside the if statement and invert it.